plugins/modules/files/iso_customize.py

```diff
@@ -0,0 +1,326 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2022, Ansible Project
+# Copyright (c) 2022, VMware, Inc. All Rights Reserved.
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: iso_customize
+short_description: Add/remove/change files in ISO file
+description:
+  - This module is used to add/remove/change files in ISO file.
+  - The module will always execute all I(delete_files) actions first, then all I(add_files) actions.
+  - We can change an existing file inside the ISO image by combining options I(delete_files)
```

Comment:
I don't think it makes sense to having to list a file both in `delete_files` and `add_files` to change it. It should *only* be listed in `add_files`.

Reply:
Thanks for your suggestion. Let me think about it.

Reply:
Thanks. change it.

Note:
The function "add_file" in pycdlib will overwrite the exiting file in ISO with type ISO9660, Rock Ridge 1.12, Joliet, UDF.
But it won't overwrite the existing file in ISO with  Rock Ridge 1.09 / 1.10
So we take workaround here: delete the existing file and then add file
    if iso_rr_check_file_exist(opened_iso, dest_file):
                opened_iso.rm_hard_link(iso_path=file_in_iso_path)

Reply:
> Note: The function "add_file" in pycdlib will overwrite the exiting file in ISO with type ISO9660, Rock Ridge 1.12, Joliet, UDF. But it won't overwrite the existing file in ISO with Rock Ridge 1.09 / 1.10 So we take workaround here: delete the existing file and then add file

This should **really** go into the documentation.

Reply:
Thanks. Fix it.